geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/DescribeMappingCommand.java

```diff
@@ -180,19 +161,8 @@ public ConfigurationPersistenceService checkForClusterConfiguration()
 
   private RegionConfig checkForRegion(String regionName, CacheConfig cacheConfig, String groupName)
```

Comment:
Do we still need this private method `checkForRegion `? Since we can call `MappingCommandUtils.checkForRegion(regionName, cacheConfig, groupName)`.

Reply:
✅